src/sql/base/test/browser/ui/scrollableView/scrollableView.test.ts

```diff
@@ -63,128 +63,141 @@ suite.skip('ScrollableView', () => { // TODO chgagnon Fix these tests
 		scrollableView.dispose();
 	});
 
-	test('adds and removes views correctly', async () => {
-		const view1 = new TestView(20, 20);
-		const view2 = new TestView(20, 20);
-		const view3 = new TestView(20, 20);
-		const scrollableView = new ScrollableView(container);
+	test('adds and removes views correctly', async (): Promise<void> => {
+		(async () => {
+			const view1 = new TestView(20, 20);
+			const view2 = new TestView(20, 20);
+			const view3 = new TestView(20, 20);
+			const scrollableView = new ScrollableView(container);
 
-		scrollableView.layout(200, 200);
+			scrollableView.layout(200, 200);
 
-		scrollableView.addView(view1);
-		scrollableView.addView(view2);
-		scrollableView.addView(view3);
+			scrollableView.addView(view1);
+			scrollableView.addView(view2);
+			scrollableView.addView(view3);
 
-		// we only update the scroll dimensions asynchronously
-		await waitForAnimation();
+			// we only update the scroll dimensions asynchronously
+			await waitForAnimation();
 
-		let viewQuery = getViewChildren(container);
-		assert.equal(viewQuery.length, 3, 'split view should have 3 views');
+			let viewQuery = getViewChildren(container);
+			assert.equal(viewQuery.length, 3, 'split view should have 3 views');
 
-		scrollableView.clear();
+			scrollableView.clear();
 
-		viewQuery = getViewChildren(container);
-		assert.equal(viewQuery.length, 0, 'split view should have no views');
+			viewQuery = getViewChildren(container);
+			assert.equal(viewQuery.length, 0, 'split view should have no views');
 
-		view1.dispose();
-		view2.dispose();
-		view3.dispose();
-		scrollableView.dispose();
+			view1.dispose();
+			view2.dispose();
+			view3.dispose();
+			scrollableView.dispose();
+		})();
 	});
 
+
 	test('shrinks views correctly', async () => {
-		const view1 = new TestView(20, Number.POSITIVE_INFINITY);
-		const view2 = new TestView(20, Number.POSITIVE_INFINITY);
-		const view3 = new TestView(20, Number.POSITIVE_INFINITY);
-		const scrollableView = new ScrollableView(container);
+		(async () => {
+			const view1 = new TestView(20, Number.POSITIVE_INFINITY);
+			const view2 = new TestView(20, Number.POSITIVE_INFINITY);
+			const view3 = new TestView(20, Number.POSITIVE_INFINITY);
+			const scrollableView = new ScrollableView(container);
 
-		scrollableView.layout(200, 200);
+			scrollableView.layout(200, 200);
 
-		scrollableView.addView(view1);
+			scrollableView.addView(view1);
 
-		await waitForAnimation();
+			await waitForAnimation();
 
-		assert.equal(view1.size, 200, 'view1 is entire size');
+			assert.equal(view1.size, 200, 'view1 is entire size');
 
-		scrollableView.addView(view2);
+			await waitForAnimation();
```

Comment:
Why did you remove the addView call?

Reply:
Sorry about that, I must have lost that line when I was tinkering with the surrounding async calls. I'll add it back.